cmd/lotus/debug_advance.go

```diff
@@ -46,6 +46,7 @@ func init() {
 					return xerrors.Errorf("StateMinerWorker: %w", err)
 				}
 
+				// XXX: This can't be right
 				rand, err := api.ChainGetRandomnessFromTickets(ctx, head.Key(), crypto.DomainSeparationTag_TicketProduction, head.Height(), addr.Bytes())
```

Comment:
We need help on this command, what is this trying to achieve?

Reply:
filename: `debug_advance.go`

it just makes a block in debug mode. where pretty much all checks are disabled

Reply:
I am not too sure what randomness need to be retrieved here and not important for my PR(?)

Reply:
So, dead code? Because this doesn't generate the correct randomness.

Reply:
it just doesnt matter, its not verified

Reply:
It needs to be anything IIRC.